src/GoogleFitOnFhir/Common/Constants.cs

```diff
@@ -26,5 +26,10 @@ public class Constants
         /// String identifier for the GoogleFit platform.  Used to help identify the platform to import from, in a <see cref="QueueMessage"/>.
         /// </summary>
         public static string GoogleFitPlatformName => "GoogleFit";
+
+        /// <summary>
+        /// Sets the results limit of a Dataset request
+        /// </summary>
+        public static int GoogleFitDatasetRequestLimit => 10;
```

Comment:
I actually think that we should move this to a new class called GoogleFitDataImporterContext. At some point in the near future, many of the hard coded values will be moving to environment variables. We have some IoC container extensions to easily deserialize environment variables into objects. We should start creating these objects now.